### PR TITLE
Changes from He-Man

### PR DESCRIPTION
**Pull request by Ignatz-HeMan**
Here some more changes from me.
I also have changed the "isbuildallowed" to a full configurable code, so you can:
- define the max number of doors
- define the max number of Gates
- define the max building heigh
- define the max building count based on the group size 
- define the max number of storages based on the group size.

I have not implemented here, because I don't know, if it is interesting for you and I have to change a bit before.
If you want it, I will do it...

### CODE REVIEW

**Comment by vbawol** [comment]
Thanks, The base building changes sound good, please go ahead with a pull request.
https://github.com/EpochModTeam/Epoch/issues/241

File: Sources/epoch_code/gui/scripts/dynamenu/Epoch_dynamicMenu.sqf [hunk 1]
```diff
@@ -108,11 +108,21 @@ _checkConfigs = {
 							format ["%2 %1 call Epoch_dynamicMenu;",_arr + [(configName _x)],getText(_x >> "action")]
 						};
 
+						_tooltip = "";
+						_tooltipcode = getText(_x >> "tooltipcode"),;
```
Comment (vbawol) [inline Sources/epoch_code/gui/scripts/dynamenu/Epoch_dynamicMenu.sqf:112]: @raymix notced you have an extra comma here, I tested and funny enough it doesn't throw an error

and neither does
EP = 0,121232434234324;

**Comment by Ignatz-HeMan** [comment]
lol, this extra comma is there since a few weeks. I am already working with tooltipcodes.
But you are right. This have to be deleted.
Thanks!

**Comment by raymix** [comment]
I like your idea of making tooltips more dynamic @Ignatz-HeMan, but be careful with call compile as it can be abused. You are loading variables from configs anyway so that shouldn't be a problem.

**Pull request merged**